e2e/ts_library/reference_types_directive/BUILD.bazel

```diff
@@ -7,7 +7,7 @@ ts_library(
     expected_diagnostics = [
         "TS2304: Cannot find name 'Hammer'",
     ],
-    node_modules = "@npm//:node_modules",
+    node_modules = "@npm//:node_modules_filegroup",
```

Comment:
this makes me think it's a breaking change. I hoped we could keep the old code building?

Reply:
Because `@npm//:node_modules` is not a node_modules_library and no longer a filegroup and the rules_typescript change makes ts_library now all of its .d.ts files and stamp out the `types` in tsconfig this ts_library now successfully compiles. This test was checking for the missing dep "TS2304: Cannot find name 'Hammer'" so for the test to work as designed the node_modules must be a plain filegroup.